lib/generator.js

```diff
@@ -160,8 +191,7 @@ const generateReturns = (comments, customTypes, usedTypes) => {
   const typeLine = generateTypeLine(comment, customTypes, usedTypes);
   const line = generateComment(comment, customTypes, usedTypes, typeLine);
   const res = generateParameters(comment.parameters, customTypes, usedTypes);
-  if (res.length === 0) return [line, ''];
-  else return [line, ...res];
+  return [line, '', ...res];
```

Comment:
Double check here, maybe `if (res.length === 0) ...` needed but I mentioned that you just do not need else because all cases after `if...return` will be automatically else cases.

Reply:
Yes, but since
```js
  if (res.length === 0) return [line, ''];
  return [line, '', ...res];
```
and
```js
  return [line, '', ...res];
```
will do the same thing, I thought that the latter one looks a bit better. WDYT?